src/main/java/org/kainos/ea/db/AuthDao.java

```diff
@@ -0,0 +1,77 @@
+package org.kainos.ea.db;
+
+import org.apache.commons.lang3.time.DateUtils;
+import org.kainos.ea.cli.Login;
+import org.kainos.ea.client.FailedToGetUserId;
+import org.mindrot.jbcrypt.BCrypt;
+import java.sql.*;
+import java.util.Date;
+import java.util.UUID;
+
+public class AuthDao {
+    private DatabaseConnector databaseConnector;
+
+    public AuthDao(DatabaseConnector databaseConnector) {
+        if(databaseConnector == null)
+        {
+            throw new NullPointerException("Database connector passed in ctor is null");
+        }
+        this.databaseConnector = databaseConnector;
+    }
+
+    public boolean validLogin(Login login) {
+        try (Connection c = databaseConnector.getConnection()) {
+            Statement st = c.createStatement();
+
+            ResultSet rs = st.executeQuery("SELECT password FROM `User` WHERE email = '"
+                    + login.getEmail() + "'");
+            while (rs.next()) {
+                return BCrypt.checkpw(login.getPassword(), rs.getString("password"));
```

Comment:
Having the password check here breaks the single responsibility principle. This function is now getting the password from the db and validating it